src/layout/MainLayout/index.js

```diff
@@ -75,11 +73,6 @@ const MainLayout = () => {
         dispatch({ type: SET_MENU, opened: !leftDrawerOpened });
     };
 
-    useEffect(() => {
-        dispatch({ type: SET_MENU, opened: !matchDownMd });
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [matchDownMd]);
-
```

Comment:
I believe this will cause to show the Navbar open on first load, maybe a bad UX in mobile

Reply:
@ct-jahidhasan please fix this and make sure it happens only when media is larger than mobile.